src/erl.rs

```diff
@@ -1215,6 +1233,11 @@ fn expr<'a>(expression: &'a TypedExpr, env: &mut Env<'_>) -> Document<'a> {
             ..
         } => const_inline(literal, env),
 
+        TypedExpr::ModuleSelect {
+            constructor: ModuleValueConstructor::Inline,
+            ..
+        } => todo!(),
```

Comment:
I think we'd render the identity function here.

```suggestion
        } => "fun(A) -> A end".to_doc()
```